package.json

```diff
@@ -149,5 +149,6 @@
       "release": true,
       "tokenRef": "GITHUB_AUTH"
     }
-  }
+  },
+  "types": "./index.d.ts"
```

Comment:
Does this not need to be `types/index.d.ts`?

Reply:
Ack, yes.  types on the root so I believe this is correct.